microprofile/openapi/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.microprofile</groupId>
+        <artifactId>helidon-microprofile-project</artifactId>
+        <version>1.1.1-SNAPSHOT</version>
+    </parent>
+    <groupId>io.helidon.microprofile.openapi</groupId>
+    <artifactId>helidon-microprofile-openapi</artifactId>
+    
+    <name>Helidon Microprofile OpenAPI</name>
+
+    <description>
+        Helidon Microprofile OpenAPI implementation
+    </description>
+
+    <packaging>jar</packaging>
+    
+    <properties>
+        <spotbugs.exclude>etc/spotbugs/exclude.xml</spotbugs.exclude>
+    </properties>
+    
+    <build>
+        <plugins>
+            <plugin>
+                <!--
+                    Used for the BasicServerTest.
+                -->
+                <groupId>org.jboss.jandex</groupId>
+                <artifactId>jandex-maven-plugin</artifactId>
+                <executions>
+                  <execution>
+                    <id>make-test-index</id>
+                    <goals>
+                      <goal>jandex</goal>
+                    </goals>
+                    <phase>process-test-classes</phase>
+                    <configuration>
+                        <classesDir>${project.build.directory}/test-classes</classesDir>
```

Comment:
This looks like a mix of 2 spaces indent and 4 spaces indent.

Reply:
Fixed